frontend/src/libs/components/sidebar/sidebar.tsx

```diff
@@ -18,13 +18,7 @@ const Sidebar: React.FC<Properties> = ({ routes }) => {
     <div className={styles['sidebar']}>
       <nav className={styles['nav']}>
         <Link to={AppRoute.ROOT}>
-          <span className={styles['logo']}>
-            <img
-              src={logoS}
-              className={styles['logo-img']}
-              alt="CalmPal Logo blue"
-            />
-          </span>
+          <img src={logoS} className={styles['logo']} alt="CalmPal logo" />
```

Comment:
You might want to use the Icon component from cp-55 and cp-61, it's a universal component for rendering svgs

Reply:
it's not for all SVGs, only for icons, so it's ok here